app.test.js

```diff
@@ -132,4 +132,108 @@ describe('Server', () => {
     });
   });
 
+  describe('GET /api/v1/palettes/:id', () => {
+    it('should return a status code of 200 and a single palette object', async () => {
+      const expectedPalette = await database('palettes').first();
+        delete expectedPalette.created_at;
+        delete expectedPalette.updated_at;
+
+      const { id } = expectedPalette;
+      const response = await request(app).get(`/api/v1/palettes/${id}`);
+      const result = response.body;
+        delete result.created_at;
+        delete result.updated_at;
+
```

Comment:
With regard to the response.body from when Supertest hits our GET a palette endpoint, I don't think we need the deletes on lines 144 & 145. Based off of the endpoint, those properties should already be excluded.

```
    response.status(200).json({
      id: palette[0].id,
      title: palette[0].title,
      color_1_id: palette[0].color_1_id,
      color_2_id: palette[0].color_2_id,
      color_3_id: palette[0].color_3_id,
      color_4_id: palette[0].color_4_id,
      color_5_id: palette[0].color_5_id,
      project_id: palette[0].project_id
    });
```

I'll go ahead with the merge and strip this code if I'm not mistaken.